tests/NovaResourceGeneratorTest.php

```diff
@@ -62,8 +62,8 @@ public function testCreateNovaTestAlreadyExists()
         ]);
 
         $this->assertExceptionThrew(
-            className: ClassAlreadyExistsException::class,
-            message: 'Cannot create PostResource cause PostResource already exists. Remove PostResource.',
+            className: ResourceAlreadyExistsException::class,
+            message: 'Cannot create App\Models\PostResource cause it already exists. Remove App\Models\PostResource and run command again.',
```

Comment:
looks strange, i suppose it should be `App\Nova\PostResource` or simething like this